ckanext/canada/controller.py

```diff
@@ -262,6 +266,40 @@ def fgpv_vpgf(self, pkg_id):
         return render('fgpv_vpgf/index.html', extra_vars={
             'pkg_id': pkg_id,
         })
+    def data_dictionary(self, pd_file):
+        names = pd_file.split('.')
+        if not (len(names)==2 and names[1]=='csv'):
+            return None
+        chromo = h.recombinant_get_chromo(names[0])
+        if not chromo:
+            return None
+
+        current_lang= get_lang()
+        set_lang('fr')
+        csv_dict = [(u'Field Name / Nom de la zone',
+            u'English Description / Description en anglais',
+            u'French description / Description en français')]
+        csv_dict += [ (f['datastore_id'], f['label'], _(f['label']))
+                for f in chromo['fields'] ]
```

Comment:
instead of `set_lang()` and `_()` use `recombinant_language_text(..., lang)` https://github.com/open-data/ckanext-recombinant/blob/master/ckanext/recombinant/helpers.py#L15. That helper will handle cases where we've added both languages directly in to the schema or where we're using gettext.

I've added a bunch of the other fields that TBS would like in the data dictionaries for the new consultations type. so you might want to add those columns too (I think they requested some of them in the ticket) https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/tables/consultations.yaml#L16-L25